src/rhsmlib/dbus/objects/register.py

```diff
@@ -241,29 +241,52 @@ def _get_owner_cb(self, owners):
         # We return None here, because we cannot know what will be selected by user
         return None
 
-    def _enable_content(self, cp, consumer: dict) -> Union[dict, None]:
+    @staticmethod
+    def _enable_content(cp, consumer: dict) -> None:
```

Comment:
since this removes `Union`, the import for it is unused (and triggers a flake8 error)